storage/Utils/SystemCmd.h

```diff
@@ -43,47 +42,79 @@ namespace storage
 
     class OutputProcessor;
 
+
     /**
-     * Class to invoke a shell command and capture its output.
-     * By default, stdout and stderr output are kept separate.
-     * Use 'setCombine()' to merge them.
+     * Class to invoke a shell command and capture its exit value and output.
      */
     class SystemCmd : private boost::noncopyable
     {
     public:
 
 	enum ThrowBehaviour { DoThrow, NoThrow };
 
+
+	/**
+	 * Class with detailed options for the SystemCmd class.
+	 */
+	struct Options
+	{
+	    Options(const string& command, ThrowBehaviour throw_behaviour = NoThrow)
+		: command(command), throw_behaviour(throw_behaviour), stdin_text(),
+		  mockup_key() {}
+
+	    /**
+	     * The command to be executed.
+	     */
+	    string command;
+
+	    /**
+	     * Should exceptions be thrown or not?
+	     */
+	    ThrowBehaviour throw_behaviour;
```

Comment:
Please add a constructor that properly initializes those member variables that don't have a reasonable default constructor - like the ThrowBehaviour.